committees/forms.py

```diff
@@ -0,0 +1,75 @@
+from django.core.exceptions import ValidationError
+from django.forms import Form, ModelForm, IntegerField
+from django.forms.widgets import HiddenInput
+
+from core.forms import MarkdownForm
+
+from committees.models import GroupExternalUrl, AssociationGroup, AssociationGroupMembership
+
+
+class DeleteGroupExternalUrlForm(Form):
+
+    def __init__(self, *args, instance=None, **kwargs):
+        self.instance = instance
+        super(DeleteGroupExternalUrlForm, self).__init__(*args, **kwargs)
+
+    def delete(self):
+        self.instance.delete()
+
+
+class AddOrUpdateExternalUrlForm(ModelForm):
+    id = IntegerField(min_value=0, required=False, widget=HiddenInput)
```

Comment:
Can add [`disabled`](https://docs.djangoproject.com/en/2.2/ref/forms/fields/#disabled) to prevent tampering by the user. (or is that already the case due to using a HiddenInput?)